modules/nf-core/proovframe/map/main.nf

```diff
@@ -8,7 +8,9 @@ process PROOVFRAME_MAP {
         'biocontainers/proovframe:0.9.7--hdfd78af_1' }"
 
     input:
-    tuple val(meta), path(faa), path(fasta)
+    tuple val(meta), path(fasta)
+    path(faa)
+    path(db)
```

Comment:
Why are you moving the faa out from the tuple?
Generally we like to have meta maps on most file inputs, because it gives you more flexibility in pipelines e.g. putting a meta on a genome fasta helps support multiple genomes in one pipeline